src/main/java/com/blackducksoftware/integration/hub/api/component/ComponentRestService.java

```diff
@@ -48,4 +50,20 @@ public ComponentRestService(final RestConnection restConnection, final Gson gson
         return allComponents;
     }
 
+    public ComponentItem getExactComponentMatch(String id, String groupId, String artifactId, String version)
```

Comment:
So the "id:" refers to the ENTIRE string, not just the first component of it. So forge, packageManager, namespace etc would cover all the "maven, rubygems, npmjs, nuget," strings. It's really just another piece OF the id the query wants to supplement the gav, right?